server/controllers/events/events_controller.go

```diff
@@ -627,7 +627,7 @@ func (e *VCSEventsController) HandleAzureDevopsPullRequestCommentedEvent(w http.
 		return
 	}
 
-	if *resource.Comment.IsDeleted {
+	if resource.Comment.GetIsDeleted() {
```

Comment:
This looks correct to me.

Were you able to build Atlantis and run it locally against your azure devops to test this out?